tools/dev-data-cleanup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# This script generates a list of IAM policies that can be removed from broad-jade-dev-data project
+# Users can follow the steps below to update the IAM policy
+# Why? Clear out unused IAM policies to avoid hitting 250 IAM policy limit
+
+#---USAGE---
+#gcloud auth login #your dev gmail
+#export token=$(gcloud auth print-access-token)
+#gcloud auth login #your broad email
+#./dev-data-cleanup.sh broad-jade-dev-data
+
+#----RESULT---
+#This generates a few files, including:
+#1. iamPolicyONLY.txt. These are the policies that should be removed.
+#2. backupPolicy.json - existing policy
+#NEXT STEPS:
+#1. Make a copy of backPolicy.json, named updatedPolicy.json
+#2. Remove group policies listed in iamPolicyONLY.txt
+#3. Update the IAM policy to use the udpated one by running:
+# gcloud projects set-iam-policy broad-jade-dev-data updatedPolicy.json
+
+#----- TROUBLESHOOTING----
+#Error: parse error: Invalid numeric literal at line 1, column 10
+# + empty samPolicies.txt files
+# REASON: your token has expired. Run the steps above again!
+
+# Require a project id
+: ${1?"Usage: $0 PROJECT_ID"}
+PROJECT_ID=$1
+
+PROJECT_EXISTS=$(gcloud projects list --filter ${PROJECT_ID} --uri 2>/dev/null)
+if [ -z "${PROJECT_EXISTS}" ]; then
+    echo "ERROR: Cannot find project '${PROJECT_ID}' (do you have permission?)"
+    exit 1
+fi
+
+#Require user to set BEARER TOKEN
+if [ -z "${token}" ]; then
+    echo "ERROR set token env variable. Try gcloud auth login and then export token=$(gcloud auth print-access-token)"
```

Comment:
This will actually evaluate `gcloud auth print-access-token`. The string should be wrapped in single-quotes to avoid this.

Reply:
Nice catch!